xchange-kucoin/src/main/java/org/knowm/xchange/kucoin/KucoinAccountServiceRaw.java

```diff
@@ -116,4 +117,49 @@ public InternalTransferResponse innerTransfer(InnerTransferRequest req) throws I
                 pageSize,
                 currentPage));
   }
+
+  public DepositAddressResponse createDepositAddress(
```

Comment:
can you keep parameters on the same line as method name for these three?

Reply:
I too prefer them in the single line. But when contributing I try to replicate existing style in the same class. Other methods (like getDepositList, getWithdrawalsList, getAccountLedgers, etc).

Reply:
ah that's a shame then leave it as is